libr/lang/p/Makefile

```diff
@@ -47,24 +47,25 @@ lang_python.${EXT_SO}:
 	${LDFLAGS_LIB} -o lang_python.${EXT_SO} python.c -lpython27
 else
 PYCFG=../../../python-config-wrapper
+PYSO=lang_python$(PYVER).${EXT_SO}
```

Comment:
`PYVER` is expected to be passed by r2pm (value is either '2' or '3') to make the shared library name (`PYSO`), my thought about using a new env instead of inferring from `PYCFG` is that in that case when uninstalling we would still need to pass `PYTHON_CONFIG` in r2pm script, so one needs to have a corresponding python-config in system to uninstall it, which seems unreasonable.